mkdocs.yml

```
@@ -40,6 +40,7 @@ nav:
     - Styles:
       - Display:                           'fundamentals/styles/display.md'
       - Flex:                              'fundamentals/styles/flex.md'
+      - Overflow:                          'fundamentals/styles/visibility.md'
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)